src/features/questions/ui/button/SendButton.tsx

```
@@ -0,0 +1,46 @@
+
+  const { mutateAsync } = usePostSendMessage()
+
+  const onSubmit = handleSubmit(async (formValues) => {
+    try {
+      const response = await mutateAsync({
+        receiverId: 1,
+        emotionId: 1,
+        content: formValues.message,
+      })
+
+      if (response && response.dmId) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)